rts/Rml/SolLua/bind/Global.cpp

```diff
@@ -276,20 +276,15 @@
 			  */
 			 "SetDebugContext", sol::overload(&functions::setDebugContext, &functions::setDebugContextByName),
  
-			 /***
-			  * Global functions for Recoil's RmlUi implementation. 
-			  * @table RmlUi
-			  * @field contexts RmlUi.Context[]
-			  * @field version string RmlUi version
-			  */
- 
 			 // G
+			 /*** @field RmlUi.contexts RmlUi.Context[] */
 			 "contexts", sol::readonly_property(&getIndexedTable<Rml::Context, &functions::getContext, &functions::getMaxContexts>),
 			 //--
+			 /*** @field RmlUi.version string RmlUi version */
 			 "version", sol::readonly_property(&Rml::GetVersion)
 		 );
 		 /***
-		  * @alias RmlUi.key_identifier
+		  * @enum RmlUi.key_identifier
```

Comment:
This one needs `@fields`. I can do this in five seconds with a Vim macro if you cbf.